src/main/java/edu/greenblitz/robotName/commands/arm/Elbow/MoveElbowToAngle/SimulationMoveElbowToAngle.java

```diff
@@ -0,0 +1,23 @@
+package edu.greenblitz.robotName.commands.arm.Elbow.MoveElbowToAngle;
+
+import edu.greenblitz.robotName.commands.arm.Elbow.ElbowCommand;
+
+public class SimulationMoveElbowToAngle extends ElbowCommand {
+
+    double goalAngle;
```

Comment:
Is there a reason for this not to be private?

If you do decide to chagne it, what's the difference between this and the "real" class

Reply:
one run in execute and one in initialize